src/shared/components/tc-communities/JoinCommunity/ConfirmModal/index.jsx

```diff
@@ -94,4 +109,10 @@ ConfirmModal.propTypes = {
   resetJoinButton: PT.func.isRequired,
   token: PT.string,
   userId: PT.string,
+  theme: PT.shape({
+    primaryBtn: PT.shape().isRequired,
+    secondaryBtn: PT.shape().isRequired,
+  }).isRequired,
 };
+
+export default themr('ConfirmModal', { primaryBtn, secondaryBtn })(ConfirmModal);
```

Comment:
I guess, when you specified `primaryBtn` and `secondaryBtn` themes here you though you pass them in as default styling, but actually because later you pass them into the buttons via their `theme` props, the button treat them as ad-hoc styles, thus breaking entire logic of context styling.